docs/src/faqs/migrate_from_fnapi.rst

```diff
@@ -233,85 +80,60 @@ The following is a collection of suggested changes to optimize Docker builds by
 
 .. caution::
 
-    **Most of these changes are incompatible with the FnAPI.**
-
-    The following changes will break your job if you return to using the FnAPI. If you choose to switch back to the FnAPI, simply undo these deletions.
-
-* **DELETE** any lines updating & installing Debian packages, i.e. ``apt-get update && apt-get install ...``. These commands have been moved to ``setup.py`` and will run via the added line ``RUN pip install .`` .
-* **DELETE** any lines referring to ``pip install -r job-requirements.txt`` and ``pip install klio-exec``; dependencies will be installed via the added line ``RUN pip install .``.
-
-    .. note::
+    **Most of these changes are incompatible with using setup.py.**
 
-        Note: Keeping ``pip install --upgrade pip setuptools`` (or similar) is still advised.
+    The following changes will break your job if you return to using ``setup.py`` to package your dependencies. If you choose to switch back, simply undo these deletions.
 
-* **DELETE** any lines creating ``/usr/src/config``, i.e. ``RUN mkdir -p /usr/src/config``.
-* **DELETE** the two lines ``ARG KLIO_CONFIG=klio-job.yaml`` and ``COPY $KLIO_CONFIG /usr/src/config/.effective-klio-job.yaml``.
+* **DELETE** lines copying ``MANIFEST.in`` and ``setup.py`` as they are no longer used.
```

Comment:
Maybe warn users that if they do not follow this suggestion, and they do not actually have  `MANIFEST.in` and `setup.py` files, that building the image will break as docker will complain that it can't find them. Or make it a required change.